src/OpenLoco/Things/Misc.cpp

```diff
@@ -35,7 +35,7 @@ exhaust* OpenLoco::exhaust::create(loc16 loc, uint8_t type)
         _exhaust->var_14 = obj->var_05;
         _exhaust->var_09 = obj->var_06;
         _exhaust->var_15 = obj->var_07;
-        _exhaust->type = misc_thing_type::exhaust;
+        _exhaust->mType = MiscThingType::exhaust;
```

Comment:
Not a fan of the abbreviations. Would rather see `vehicleType` and `miscType`. Or a `getSubType()` and `setSubType()` method, with the `subType` being treated as a 'private' `uint8`. I think the latter has my preference.

Reply:
okay what about my new version?